src/plugin.ts

```diff
@@ -0,0 +1,36 @@
+import { SitemapContext } from "./sitemap.ts";
+import { Manifest, Plugin } from "./types.ts";
+
+interface PluginOptions {
+	include: string[];
+	exclude: string[];
+}
```

Comment:
Could we make Options optional?
Users may not need to set them
```suggestion
interface PluginOptions {
	include?: string[];
	exclude?: string[];
}
```